### ISSUE TITLE
Add code to create pseudo ensemble for snow DA

### ISSUE THREAD

**Issue by CoryMartin-NOAA** [NOAA-EMC/GDASApp#199]
- [x] Add a python script to create a pseudo ensemble
- [x] Add a ctest verifying this test works as intended

**Comment by jiaruidong2017** [comment]
@CoryMartin-NOAA For this task, I did and will do the following steps. Please correct me if you think some steps are not accurate. Thanks.

1) I tested the python script (`letkf_create_ens.py`) offline under the GDASApp environment, and it successfully generated a pseudo ensemble at both C96 and C48 resolutions. 

2) For the next step, I put the python script in the directory `ush/land/letkf_create_ens.py` for future use. 

3) Add a ctest in the test/land/CMakeLists.txt as following: 

```
# test for create a letkfoi ensemble
add_test(NAME test_gdasapp_letkfoi_create_ens
         COMMAND ${PROJECT_SOURCE_DIR}/test/land/create_bkg_ens.sh ${PROJECT_BINARY_DIR} ${PROJECT_SOURCE_DIR}
         WORKING_DIRECTORY ${PROJECT_BINARY_DIR}/test/land/)
```

A test script `create_bkg_ens.sh` is put at `test/land/` directory to prepare and initiate this ctest.

**Comment by CoryMartin-NOAA** [comment]
@jiaruidong2017 I think this all seems to be correct, now we just need to stage the input files somewhere (either on disk or in a tarball depending on size) and then I think this is ready for a PR/review.

**Comment by CoryMartin-NOAA** [comment]
This should have been closed by #207
